tests/Unit/Parallel/Test_GlobalCache.cpp

```diff
@@ -229,6 +229,12 @@ void TestArrayChare<Metavariables>::run_test_one() noexcept {
   SPECTRE_PARALLEL_REQUIRE(
       6 == Parallel::get<animal_base>(local_cache).number_of_legs());
 
+  // test the serialization of the caches
```

Comment:
how does this test the serialization of the cache?

Reply:
oops -- I put this in the wrong commit and on the wrong line. I've moved it to the `Add pup function for GlobalCache pointer` commit, and on the line that introduces the first serialization test.